<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Flavivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flavivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flaviviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/zipball/master" class="btn">Download</a>
      <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous flavivirus (EVE) data</strong>
	   </h3>
	   <hr>

		<p>
          We have used GLUE to organise the genomic 'fossil record' of flaviviruses.
          This page provides a description of Flavivirus-GLUE's paleovirus component, and
          quick links to specific data items. 
          
		</p>

        <p><img src="../assets/images/eve-header.png" alt="Species with endogenous flaviviruses" /></p>

<blockquote>
		<p>
		  Some of the species in which we identified novel <b>endogenous flaviviral elements (EFVs)</b>
		  Left to right: 
		  freshwater jellyfish (<i>Craspedacusta sowerbyi</i>),
		  long-horned beetle (<i>Anoplophora glabripennis</i>),
		  tadpole shrimp (<i>Lepidurus arcticus</i>),
		  tube-eye fish (<i>Stylephorus chordatus</i>).
		</p>
</blockquote>
		 


	   <br>
	   <h3>
		 <a id="Virology" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to virology</strong>
	   </h3>        
	   <hr>

	   <p>
          Endogenous viral sequences can inform our understanding of contemporary viruses
          in a wide variety of ways. Perhaps most importantly, EVEs allow calibration of
          the long-term evolutionary history of virus groups, which greatly influences 
          how we understand their biology.
 	   </p>

	   <p>
         Importantly, once time calibrations have been established, a far richer range of
         comparative genomic studies can be performed. By examining
         variation in the light of a known evolutionary history, these studies can provide 
         invaluable insights into the biological mechanisms through which viruses 
         replicate and spread.
	   </p>

	   <p>
          In some cases, studies of paleoviruses can go a step further -
          EVE sequences can be used to guide the reconstitution of functional nucleic acids
          and proteins via gene synthesis, so that their biological properties
          may be empirically investigated. This in turn, can greatly expand the scope of 
          comparative/theoretical approaches in paleovirology.
	   </p>


         
	   <br>
	   <h3>
		 <a id="Metagenomics" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to viral metagenomics</strong>
	   </h3>
       <hr>


	   <p>
          The endogenous flaviviral element (EFV) sequences in Flavivirus-GLUE can
          provide a useful resource for those
          interested in identifying and characterising flaviviruses in metagenomic
          datasets.
	   </p>


	   <p>
          Firstly, these sequences be used to exclude any potential 'false positive'
          hits (i.e. sequences that seem to represent new flaviviruses but in fact derive
          from genomic DNA).
	   </p>


	   <p>
          In addition, when new flavivirus species are identified, inclusion of EVEs
          in phylogenetic analyses can often provide useful information
          about their broader ecology and evolution, including (uniquely) their long-term
          evolution. 
	   </p>
        

       
	   <br>
	   <h3>
		 <a id="Genomics" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to genomics</strong>
	   </h3>        
	   <hr>

	   <p>
          EFVs are not only useful genetic markers, several lines of evidence indicate
          they may have, or have had, functional roles as host alleles. The prevalence of 
          multicopy EFV lineages in some species suggests that germline incorporation of
          flavivirus sequences might have influenced the evolution of host genomes in
          important ways.
	   </p>



        
 

	   <br>	   
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for EVEs</strong>
	   </h3>
	   <hr>
	   

		<p>
          We have applied a systematic approach to naming EVE, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>

 
        <p><img src="../assets/images/efv-nomenclature.png" alt="EFV Nomenclature" /></p>

 		
	

		<p>
          The first component is the classifier ‘EFV’ (endogenous flavivirus). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of EVE group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer that identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
		</p>
		
		<p>
          The third component of the ID defines the set of host species in which the ortholog occurs.
 		</p>




	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of Flavivirus-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of EVE-specific data.
		   These schema extensions are defined in 
           <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/eve/efvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional table: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains EVE locus information: e.g. species, assembly, scaffold, location coordinates.
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains summary information for individual
		   EVE insertions. It refers to the reference sequences constructed to represent
		   each insertion, which reflect our best efforts to reconstruct progenitor virus
		   sequences as they might have looked when they initially integrated into 
		   the germline of ancestral species. 
		</p>




	   <br>
	   <h3>
		 <a id="Raw EVE sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw EVE sequences and data</strong>
	   </h3>
	   <hr>
	   

		<p>
          Raw FASTA for EFVs recovered via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>
          are <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/sources/fasta-digs-eve"><strong>here</strong></a>.
 		</p>



		<p>
          Raw data in tabular format are <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/eve/efv-side-data.tsv"><strong>here</strong></a>.
          The tabular files contain information about the genomic location of each EVE. 
  		</p>



	   <br>

	   <h3>
		 <a id="EVE reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EVE reference sequences and data</strong>
	   </h3>

	   <hr>


		 <p>
			We constructed reference sequences for EVEs using alignments of EVE
			sequences derived from the same initial germline colonisation event - i.e.
			orthologous elements in distinct species, and paralogous 
			elements that have arisen via intragenomic duplication of EVE sequences.
		 </p>



		<p>
          EFV consensus/reference FASTA is
          <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/sources/fasta-refseqs-eve"><strong>here</strong></a>.
 		</p>



		<p>
          Tabular formatted metadata for EFV reference sequences is
          <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/eve/efv-refseq-side-data.tsv"><strong>here</strong></a>.
  		</p>



	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>

		<p>

          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          ‘<a href="./alignment tree concept small.png">alignment tree</a>’
          data structure. Alignments in the project include:

		  <ol>
			<li> ‘<a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/"><strong>'Internal node' level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirus genera and EVE reference sequences.
			<li> ‘<a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/tip"><strong>'Tip' level</strong></a>’
			alignments in which all taxa are derived from a single EVE lineage.</li>
		  </ol>

 		</p>

	   <br>

	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/genus"><strong>Genus-level</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/tip"><strong>EVE lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
		Bamford CGG, de Souza WM, Parry R and RJ Gifford
		<b>(2021)</b>
		<br>
		Comparative analysis of genome-encoded viral sequences reveals the evolutionary history of the <i>Flaviviridae</i>.
		<br>
		preprint <a href="https://www.biorxiv.org/content/10.1101/2021.09.19.460981v1" target="_blank">[view]</a> 
		<br>
		<br>

		<p>
		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		Zhu H, Dennis T, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		<a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		<br>
		<br>

		Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		<b>(2018)</b>
		<br>
		Nomenclature for endogenous retrovirus (ERV) loci.
		<br>
		<i>Retrovirology</i>
		<a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		<br>
		<br>

		Katzourakis A. and RJ. Gifford
		<b>(2010)</b>
		<br>
		Endogenous viral elements in animal genomes.
		<br>
		<i>PLoS Genetics</i>
		<a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/">Flavivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



