<!DOCTYPE html>
<html lang="en-us">

  <head>


    <meta charset="UTF-8">
    <title>Flavivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flavivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flaviviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./news.html" class="btn">News</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/zipball/master" class="btn">Download</a>

      <a target="_blank" href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master" class="btn">GitHub</a>
    </section>

    <section class="main-content">


		<p>
          This tutorial page aims to illustrate some of Flavivirus-GLUE's functionalities
          via brief, command line-based examples.
          It assumes that you have already installed GLUE and built the Flavivirus-GLUE project.
	   </p>
	   

	   <h3>
		 <a id="ExampleExportProteinFasta" class="anchor" href="#ExampleExportProteinFasta" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Example 1: Exporting protein and polypeptide sequences</strong>
	   </h3>
	   <hr>


		 <p>
		   To begin, open a terminal window, start command line GLUE, and navigate into 
		   the Flaviviridae-GLUE project, as shown below: 		   
		 </p>


<pre>
<code>MyComputer:Flavivirus-GLUE rob$ gluetools.sh

GLUE Version 1.1.107
This program comes with ABSOLUTELY NO WARRANTY. This is free software, and you
are welcome to redistribute it under certain conditions. For details see
GNU Affero General Public License v3: http://www.gnu.org/licenses/

Mode path: /
GLUE> project flaviviridae 
OK
Mode path: /project/flaviviridae
GLUE></code></pre>


		 <p>
		   In GLUE, protein-coding features are explicitly defined on reference sequences
		   only. 
		   Protein-coding features of reference sequences can be exported using a 
		   'fastaProteinExporter' module as follows. 		   
		 </p>
		 

		 <p>
		   To quickly create this GLUE module with a default specification we will use the
		   -t (--template) option, as shown below: 
		   </p>

<pre>
<code>Mode path: /project/flaviviridae
GLUE> create module -t fastaProteinExporter
OK
(1 Module created)
Mode path: /project/flaviviridae
</code></pre>

		 <p>
		   The module can now be used to export NS5 protein sequences for all flavivirus
		   genus reference sequences in which the NS5 protein has been annotated.
		   One way to do this is to first navigate into the project
		   (updating the current <b>mode path</b>)
		   and then use it's
		   native export command, as shown below: 		   
		 </p>


	   
<pre>
<code>Mode path: /project/flaviviridae
GLUE> module fastaProteinExporter
OK

Mode path: /project/flaviviridae/module/fastaProteinExporter
GLUE> export reference -f NS5 -w "sequence.genus = 'Flavivirus'" -p
</code></pre>

		 <p>
		   The -f (--featureName) option is used to specify a protein coding feature.
		 </p>

		 <p>
		   The -w (--whereClause) option is used to define a 'where' clause for the command,
		   selecting specific reference sequences based on their properties.
		 </p>
		 
		 <p>
		   The -p (--preview) option sends the result of the export command to the
		   terminal screen.
		 </p>
		 
		 <p>
		   A portion of the command output is shown below. 		   
		 </p>
	   
<pre>
<code>>REF_APOIV.ncbi-refseqs.NC_003676
GVSSSYITYGEQWKRELNKLNAQAFFLYKSRLVHEIDRAEAVSNLSKGRTNTGHAVSRGTSKLAWMHERG
YVPLKGVVVDLGSGRGGWSYYAAAQERVRKVNAYTLATTKGHEQPRLVQSYGWNLVTFKKADVRTIEPYP
VDTLLCDIGESSSSSSIEAKRTLQIMDVVGKWLEISPGASFCIKILCPYNPEVIEHLSRWQHQFGGGLVR
VPHSRNSTHEMYFVSGGGGNLMSSITAVTTQLMRRFTLEAGPRHVFDINLGVGTRSNLMEKSEADKSLIA
DRITIIQNENKASWHQDPNQPYRTWTYHGSYSIRDVGTSASAPNHVVKLLAWPWLKIEKVVLMAMTDTTA
FGQQRVFKEKVDTKAPEPPKEVKKVMRLVFRWLLNHIKSKGAVVRRCTKEEFINKVNSNASIGAYLREMG
EWSSAKEAVSDPKFWNMVDKERQLHLKGKCHNCVYNLMGKREKKPGEFGVAKGSRTIWYMWLGSRFLEFE
ALGFLNEERWASRDISGGGVEGIGINYLGYHLEKMARKVGVLYADDTAGWDTRVTQADLEDERELLHFME
GEHKRLAEAIFKLTYENKVALCPRPGSKGGTVMDVISRKDQRGSGQVVTYALNTLTNIKVQLVRMAESEG
ILTPELEDLGIEQWLKQNGEDRLSRLLVSGDDCVVNALDERFGKALTWLNTMEKTRKDIEAWKPSRAYRT
WQEVEFCSHHFHELFLRDGRKLIVPCRDQHELVGRSMVSQGGSSGVAGTACLAKAYAQMWLMSYFHRRDL
RTLGFAIMSSVPKDWFPTGRTTWSVHANHEWLSNENILEIWNRVWIEENPFMEDKTTVAEWKNIPYLQRN
QDLSCSSLIDNPTRAQWAKLLKGAVMKVREMIGREHYSDYLSNMGRYQEGTEEFHMW
>REF_ccCFV.ncbi-refseqs.MK473878
GFGNVDLAGEQWKNALNQMSSSSFNKYRTHKVVERKKEQSVSRGAHKMDEIIRKFGWVPKGDVIDLGCGQ
GGWSQRMAVENEVDTVRGHTIGGTHAAPSSIKTYGHESITLFDHEDVFTKELSRKDTIVCDIGEQDNNWK
IEAQRTAKVIELVMRWLSFNPTARFCFKVLCPYDTNVMQRLRNLQRKYGGRLIRLTGSRNSNHEMYMISG
QSCSLINSVATTTIMLQNRMNENWKEKILLEDITYPEGEKSIVEPSFPPESAVKNRIKYIKATHPLIKHD
...
</code></pre>

		 <p>
		   To export to a file (rather than previewing on the console), use
		   the -f (--file) option in place of -p and specify a file name,
		   as follows:
		 </p>
		 
	   
<pre>
<code>Mode path: /project/flaviviridae/module/fastaProteinExporter
GLUE> export reference -f NS5 -w "sequence.genus = 'Flavivirus'" -f flavivirus-ns5-aa.fasta
</code></pre>


		 <p>

           This will generate a file called 'flavivirus-ns5-aa.fasta'
           - by default in the directory from which the GLUE command line was initiated -
           containing the fasta-formatted output of the export command.  
           
		 </p>
Example1Question1a

	   <h3>
		 <a id="Example1Question1a" class="anchor" href="#Example1Question1a" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Q1a: How do I know which modules, features and properties have been defined?</strong>
	   </h3>
	   <hr>
	   
		 <p>
           It is strongly recommended to use
           <a target="_blank" href="https://en.wikipedia.org/wiki/Command-line_completion"><b>command line completion</b></a> (<b>tab completion</b>)
           when working in GLUE's command line interface.
           For example, when typing the export command
           shown above, pressing the tab button once after typing the '-f' (--featureName) option
           will cause the GLUE terminal to display the available options, as shown here:
           
		 </p>
		 
<pre>
<code>Mode path: /project/flaviviridae/module/fastaProteinExporter
GLUE> export reference -f 
2K                       F                        
N-proteinase             NS1                      
NS3                      NS5                      
NS5-like                 NS5A                     
NS5B                     Npro                     
RNAse                    core                     
e-rns                    e1_hepaci                
e1_pesti                 e2_hepaci                
e2_pesti                 envelope                 
fifo                     flavi-NS2A               
flavi-NS2B               flavi-NS4A               
flavi-NS4B               flavi-capsid             
flavi-envelope           hepaci-NS2               
hepaci-NS4A              hepaci-NS4B              
hepaci-p7                membrane                 
p7                       pesti-NS2                
pesti-NS4A               pesti-NS4B               
pesti-capsid             pesti-p7                 
precursor_polyprotein    premembrane              
s_peptide                seg1-NS5                 
seg3-NS3                 x_peptide                
</code></pre>

	   
		 <p>
           Another way to investigate which modules, features and properties have been
           defined in a GLUE project is to use 'list' commands in project mode, as shown below:
           
		 </p>

	   <h3>
		 <a id="Example1Question1b" class="anchor" href="#Example1Question1b" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Q1b: How do I export protein sequences for all sequences?</strong>
	   </h3>
	   <hr>

		 <p>

           You may be thinking that it is somewhat limiting to only be able to export
           polypeptide coding sequences from reference sequences in which the locations
           of those features have been explcitly defined, and of course this is true.
           
		 </p>

		 <p>

           The fastaProteinExporter module can be used to export the polypeptide sequences
           of coding features from reference sequences in which they have not been
           explicitly defined, providing a constrained multiple sequence alignment (MSA)
           containing an annotated reference sequence.
           
		 </p>

	   
<pre>
<code>export member AL_GENUS_Flavivirus -r REF_MASTER_YFV -f NS5 -a -p           
</code></pre>

		 <p>
		   A portion of the command output is shown below:		   
		 </p>
	   
<pre>
<code>>AL_GENUS_Flavivirus.ncbi-refseqs.LC582740
GGHMGPTLGEVWKKRLNAMSKHRFYEYKTAGIVEVNRTEAREALKRGLTDHGHPVSRGAAKLHWLVERGF
LEPKGWVTDLGCGRGGWSQYTATLPQVTRVVGYTLGTGHEKPIPTESYGHNIITYHDKVDVMRLPTHLCD
TVLCDIGESSSNVDVEAGRTIAVIEMFANWLEANPGQFCCKVLCPYHPDVLAALEILVREHGGGLVRVPL
SRNSTHEMYWVSGVKNKPTRAVNELSSFLLGRMAKNHEKEIIEEVWLGTGVRHTVSEAEPDDPESIKGRL
DRIRRTHEKTWRVDTHHPYRYWNYHGSYEGAQRGSAGSLVNGVVKMLSWPWSTIESVTCMNMTDTTPFGQ
QRVFREKVDTKVPEPSPGTRKAMAVVAEWMWERFHRKVARLCTREEFVAKVHSNAAIGAWDPQMGDWSSA
AEAVQDETFWKMVDEERCRHRRGECEMCVYNMMGKREKKPAKFGKARGSRAIWYMWLGSRFLEFEALGFL
NEDHWMDRENSRAGVEGLGVQYLGYVLRDMDRLPGVGYVADDTAGWDTRVSNADLEDEEQLLRWLSPTHR
ALAEPIMKMAYHNKVALVSRRANDGLVTDVISRRDQRGSGQVVTYALNTWTNIKVQLLRMAEAEGILGDN
LLHQARVLPLRNWLARHGEERMSRMAISGDDCVVRPPSEEFAFALKALNDMGKVRKDKPEWEPSDIYAGW
EDVPFCSHHFHELVLQDGRQIVVPCRDQDELVGRARIAPGAGWSVAETACLAKAYAQMWLLLYPHRRDLR
LLAFAVCSAVPVNWMPTGRTTWSVHATREWMTNEDMLDVWNQVWIEDNPWMEEKTRVRHWNQVPYLPKSS
DIACGSLIGRVNRAAWARSIRDSVSKVRAKLGAERYVDYLGAMGRYVVQQSCYHTEVL
>AL_GENUS_Flavivirus.fasta-refseqs-eve.EFV-cISF.2-AedAlb
LGVKWKSMLNALDKDAFEKYKVRGVNETGRGDYVSRGGLKLEIIHKYKWAPIGHVVDLGCGRGGWSQRAV
MEETVTSVQGYTIGGAEKENPQRFLTRGYNLATLKAGVDVYRLAPTRCDTIMCDPNLTKEKTRTMKVLQV
LESWLAFNPDQFVCKVLSPYSLEVLRKIETLQQRHKGRLVRLAYSRNSTAEMYYISGNRSNVVRDSYMTM
AALVGRFSRHLDSVFHPEPVLPMGTRAPAASVSAMDTTNMADRVEKVRLENQATWFTDPDHPYKSFCYFG
SFVTDDVKVGGQAVNPLVRKLMWPWETVTSVVSFLMTDVSTYSQQKVLREKVDTVIPPPPPQIRLVNKAI
TRHFVRIFKEKNPRIMTKEEFIANVRNDAAIGSWSRDIPWRDVQEVVQDQSFWNLVGRERALHLQGKCNM
CLYNTMGKKEKKPGVAGEAKGSRTIWYMWLGSRFLEYEALAFLNADHWVSRENFPGGVGGVGVNYFGNYL
KEISCKGKYLFADDIAGWDTKVSSEDLEDEETLLTELSPFHKALISATMKLTYQNKVVRVQRPAKNGTVM
DVISRRDQRGSGQVGTYGLNTFTNMEAQLIRQMESEGIVSPSELENLAERVLDRWLEKYGVERLKRMAIS
GDDCVVKPIDDRFATALTALNDMGKVRKDIPQWEPSKGWNDWQQVPFCSHHFHQLIMKDGREIVVPCRNQ
DELVGRARVSQGAGWSLRETACLGKSYAQMWQLMYFHRRDLRLAANAICSAVPVDWVPTSRTTWSIHAHH
QWMTTEDMLSVWNRVWIEENPWMATKEPVMSWDEIPYLHKKQDIACGSLIG
>AL_GENUS_Flavivirus.fasta-refseqs-eve.EFV-cISF.3-AnoMin
SSNKGDALGMKWKRVLNSLTQKQFEAYRSRSVDETPRGDYVSRGGLKMREIMEVHGWEPNGKVIDLGCGR
GGWSQHLAMDRRVTEVRGYTLGGSSRENPEIFMTYGHNLCVLKPLVDVYKLEPHVTNTVICDIGESDPSA
AVERTRTLKVLALLEDWLAVSKGNFVCKVLSPYHSDVLKKLETLQHAYGGRLVRLRLSRNSTAEMYYISG
PRANMVKAVYATLRSLIGRFSSHDTPFEALAPVLPTGTRAPKAKAKAADQSVLSRRIQKLRQENIETWFN
DAENPYTSFSYHGSFVT
>AL_GENUS_Flavivirus.fasta-refseqs-eve.EFV-cISF.4-AnoSin
>AL_GENUS_Flavivirus.fasta-refseqs-eve.EFV-cISF.5-TipOle
RLEKIKDENRNTWFFDEETPYRSFKYLGSFATKKRNPEDQGVNQLIKRM*RPWKKREEVT*FMLTDVSTM
MQQEVLREKVDTVSPEPTKHIKQINRMI
...
</code></pre>



	   <h3>
		 <a id="Example1Question1c" class="anchor" href="#Example1Question1c" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Q1c: How can I format the headers of exported sequences?</strong>
	   </h3>
	   <hr>



		 <p>
		   In the example, we used a default version of the fastaProteinExporter module
		   with a basic configuration.
		   This default configuration can be viewed as follows:
		       
		 </p>
	   
<pre>
<code>Mode path: /project/flaviviridae/module/fastaProteinExporter
GLUE> show configuration 
&lt;?xml version="1.0" encoding="UTF-8" standalone="no"?&gt;
&lt;fastaProteinExporter/&gt;</code></pre>

	   <h3>
		 <a id="Example1Question1d" class="anchor" href="#Example1Question1d" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Q1d: How can I export aligned coding sequences?</strong>
	   </h3>
	   <hr>


		 <p>

           A 'fastaProteinAlignmentExporter' module can be used to export aligned protein sequences from GLUE, 
           following the same basic principles as described for the 'fastaProteinExporter'
           module above.
		       
		 </p>

<pre>
<code>Mode path: /project/flaviviridae
GLUE> create module -t fastaProteinAlignmentExporter
OK
(1 Module created)
Mode path: /project/flaviviridae

GLUE> export AL_GENUS_Flavivirus -r REF_MASTER_YFV -f NS5 -a -p</code></pre>


	   <h3>
		 <a id="Example1Question1e" class="anchor" href="#Example1Question1e" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Q1e: How do I export protein sequences for all sequences?</strong>
	   </h3>
	   <hr>

		 <p>

           All genome features, including protein-coding features, can only be identified
           within sequences if they have been explicitly annotated, or are contained in 
           an alignment. 
           Not all sequences in Flavivirus-GLUE are defined as reference sequences, but 
           all sequences are included in alignments. 
		       
		 </p>
		 
		 <p>

           However, in Flavivirus-GLUE we have used an alignment tree data structure
           to link alignments at different taxonomic levels.
           Tip-level alignments contain all sequences belonging to the corresponding
           taxonomic groups. 
		       
		 </p>

	   <h3>
		 <a id="Example1Question1f" class="anchor" href="#Example1Question1f" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Q1f: What about nucleotide-level sequences?</strong>
	   </h3>
	   <hr>

		 <p>

           GLUE's 'fastaExporter' and 'fastaAlignmentExporter' modules can be used to
           export nucleotide sequences from GLUE, following the same basic principles as
           described for the 'fastaProteinExporter' and 'fastaProteinAlignmentExporter'
           modules above.
		       
		 </p>



	   <h3>
		 <a id="ExampleAlignments" class="anchor" href="#ExampleAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Example 2: Working with alignments</strong>
	   </h3>
	   <hr>


		 <p>

           In GLUE two distinct 'types' of <b>multiple sequence alignment (MSA)</b> are recognised
           - <b>constrained</b> and <b>unconstrained</b>.
           In an unconstrained alignment the genomic coordinate space is allowed to expand
           to accomodate all insertions in any alignment member.
           In a constrained alignment, the genomic coordinate space is constrained to that of 
           a defined reference sequence - indels are effectively 'stripped out', but since
           constrained alignments are always derived from unconstrained ones, indel information
           is not lost.
		       
		 </p>

		 <p>

           To see the alignments currently contained in the Flavivirus-GLUE project
           use a 'list' command in project mode, as follows:
		       
		 </p>

<pre>
<code>Mode path: /project/flaviviridae
GLUE> list alignment 
+==============================+============================+==============================+
|             name             |        parent.name         |       refSequence.name       |
+==============================+============================+==============================+
| AL_EJP-Jingmen.1-Chironomus  | AL_GENUS_Jingmenvirus_Seg1 | REF_EJP-Jingmen.1-Chironomus |
| AL_FLAVI_SUBGENUS_cISF       | AL_GENUS_Flavivirus        | REF_KRV                      |
| AL_FLAVI_SUBGENUS_crustacean | AL_GENUS_Flavivirus        | REF_ccCFV                    |
| AL_FLAVI_SUBGENUS_dISF       | AL_GENUS_Flavivirus        | REF_LAMV                     |
| AL_FLAVI_SUBGENUS_MBFV1      | AL_GENUS_Flavivirus        | REF_DEN1                     |
| AL_FLAVI_SUBGENUS_MBFV2      | AL_GENUS_Flavivirus        | REF_MASTER_YFV               |
| AL_FLAVI_SUBGENUS_NKV1       | AL_GENUS_Flavivirus        | REF_APOIV                    |
| AL_FLAVI_SUBGENUS_NKV2       | AL_GENUS_Flavivirus        | REF_SOKV                     |
| AL_FLAVI_SUBGENUS_TBFV       | AL_GENUS_Flavivirus        | REF_POWV                     |
| AL_GENUS_Flavivirus          | AL_MINOR_FlaviTamana       | REF_MASTER_YFV               |
| AL_GENUS_Hepacivirus         | AL_MAJOR1_HepaPegi         | REF_HEPACI_1_HCV             |
| AL_GENUS_Jingmenvirus_Seg1   | AL_MAJOR2_FlaviLike        | REF_JMTV_Seg1                |
| AL_GENUS_Jingmenvirus_Seg3   | AL_MAJOR2_FlaviLike        | REF_JMTV_Seg3                |
| AL_GENUS_LGF                 | AL_MAJOR2_PestiLike        | REF_SLV2                     |
| AL_GENUS_Pegivirus           | AL_MAJOR1_HepaPegi         | REF_PEGI_1_HPgV2             |
| AL_GENUS_Pestivirus          | AL_MAJOR2_PestiLike        | REF_PESTI_1_BVDV1            |
| AL_GENUS_Tamanavirus         | AL_MINOR_FlaviTamana       | REF_TABV                     |
| AL_HEPACI_SUBGENUS_1         | AL_GENUS_Hepacivirus       | REF_HEPACI_1_HCV             |
| AL_HEPACI_SUBGENUS_2         | AL_GENUS_Hepacivirus       | REF_HEPACI_2_HCV_L           |
| AL_EFV-cISF.1-AedAeg         | AL_FLAVI_SUBGENUS_cISF     | REF_EFV-cISF.1-AedAeg        |
| AL_EFV-cISF.2-AedAlb         | AL_FLAVI_SUBGENUS_cISF     | REF_EFV-cISF.2-AedAlb        |
| AL_EFV-cISF.5-TipOle         | AL_FLAVI_SUBGENUS_cISF     | REF_EFV-cISF.5-TipOle        |
| AL_EFV-Flavi.1-CraSow        | AL_GENUS_Flavivirus        | REF_EFV-Flavi.1-CraSow       |
| AL_EFV-Flavi.2-DapMag        | AL_GENUS_Flavivirus        | REF_EFV-Flavi.2-DapMag       |
| AL_EFV-Flavi.3-LepArc        | AL_GENUS_Flavivirus        | REF_EFV-Flavi.3-LepArc       |
| AL_EFV-Tamana.3-AmpSul       | AL_GENUS_Tamanavirus       | REF_EFV-Tamana.3-AmpSul      |
| AL_EFV-Tamana.4-StyCho       | AL_GENUS_Tamanavirus       | REF_EFV-Tamana.4-StyCho      |
| AL_EFV_Tamana.2-LauKoh       | AL_GENUS_Tamanavirus       | REF_EFV-Tamana.2-LauKoh      |
| AL_EFV_X1                    | AL_MAJOR1_FlaviPesti       | REF_EFV-X1.1-PriCau          |
| AL_EFV_X2                    | AL_MAJOR2_FlaviLike        | REF_EFV-X2.2-StyCho          |
| AL_EFV_X2.1-AusLim           | AL_EFV_X2                  | REF_EFV-X2.1-AusLim          |
| AL_EFV_X2.2-StyCho           | AL_EFV_X2                  | REF_EFV-X2.2-StyCho          |
| AL_EFV_X2.3-EurAff           | AL_EFV_X2                  | REF_EFV-X2.3-EurAff          |
+==============================+============================+==============================+
Alignments 1 to 34 of 55 [F:first, L:last, P:prev, N:next, Q:quit]
</code></pre>


	   <h3>
		 <a id="ExamplePhylogenies" class="anchor" href="#ExamplePhylogenies" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Example 3: Working with phylogenies</strong>
	   </h3>
	   <hr>



		 <p>

           Phylogenies can be created using GLUEs raxmlPhylogenyGenerator module.
		       
		 </p>


	   
<pre>
<code>&lt;?xml version="1.0" encoding="UTF-8" standalone="no"?&gt;
&lt;raxmlPhylogenyGenerator&gt;
    &lt;raxmlPhylogenyRunner&gt;
        &lt;substitutionModel&gt;PROTGAMMAGTR&lt;/substitutionModel&gt;
        &lt;bootstrapReplicates&gt;100&lt;/bootstrapReplicates&gt;
    &lt;/raxmlPhylogenyRunner&gt;
&lt;/raxmlPhylogenyGenerator&gt;</code></pre>



		 <p>

           Phylogenies can be created from amino acid level MSAs using the module
           configured in this way, as shown below:
		       
		 </p>

	   
<pre>
<code>generate amino-acid phylogeny AL_TREE_Pestivirus -s -o pesti.tre NEWICK_BOOTSTRAPS</code></pre>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/">Flavivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



