<!DOCTYPE html>
<html lang="en-us">

  <head>


    <meta charset="UTF-8">
    <title>Flavivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flavivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flaviviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/zipball/master" class="btn">Download</a>

      <a target="_blank" href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">


		<p>
          This tutorial page aims to illustrate some of Flavivirus-GLUE functionalities
          via brief examples.
	   </p>

		<p>
          It assumes that you have already installed GLUE and built the Flavivirus-GLUE project.
	   </p>
	   

	   <h3>
		 <a id="ExampleExportProteinFasta" class="anchor" href="#ExampleExportProteinFasta" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Exporting protein and polypeptide sequences</strong>
	   </h3>
	   <hr>

		 <p>
		   In GLUE, protein-coding features are explicitly defined on reference sequences
		   only. 
		   Protein-coding features of reference sequences can be exported using a 
		   'fastaProteinExporter' module as follows. 		   
		 </p>
		 

		 <p>
		   First create the module. We will use the -t option to create the module 
		   with a default specification. 		   
		 </p>

<pre>
<codeMode path: /project/flaviviridae
GLUE>create module -t fastaProteinExporter
OK
(1 Module created)
Mode path: /project/flaviviridae
</code></pre>

		 <p>
		   The module can now be used to export NS5 protein sequences for all flavivirus
		   genus reference sequences in which the NS5 protein has been annotated. 		   
		 </p>


	   
<pre>
<code>Mode path: /project/flaviviridae
GLUE> module fastaProteinExporter
OK

Mode path: /project/flaviviridae/module/fastaProteinExporter
GLUE> export reference -f NS5 -w "sequence.genus = 'Flavivirus'" -p
</code></pre>

		 <p>
		   The -f (--feature) option is used to specify a protein coding feature.
		 </p>

		 <p>
		   The -w (--where) option is used to define a 'where' clause for the command,
		   selecting specific reference sequences based on their properties.
		 </p>
		 
		 <p>
		   The -p (--preview) option sends the result of the export command to the
		   terminal screen.
		 </p>
		 
		 <p>
		   A portion of the command output is shown below. 		   
		 </p>
	   
<pre>
<code>>REF_APOIV.ncbi-refseqs.NC_003676
GVSSSYITYGEQWKRELNKLNAQAFFLYKSRLVHEIDRAEAVSNLSKGRTNTGHAVSRGTSKLAWMHERG
YVPLKGVVVDLGSGRGGWSYYAAAQERVRKVNAYTLATTKGHEQPRLVQSYGWNLVTFKKADVRTIEPYP
VDTLLCDIGESSSSSSIEAKRTLQIMDVVGKWLEISPGASFCIKILCPYNPEVIEHLSRWQHQFGGGLVR
VPHSRNSTHEMYFVSGGGGNLMSSITAVTTQLMRRFTLEAGPRHVFDINLGVGTRSNLMEKSEADKSLIA
DRITIIQNENKASWHQDPNQPYRTWTYHGSYSIRDVGTSASAPNHVVKLLAWPWLKIEKVVLMAMTDTTA
FGQQRVFKEKVDTKAPEPPKEVKKVMRLVFRWLLNHIKSKGAVVRRCTKEEFINKVNSNASIGAYLREMG
EWSSAKEAVSDPKFWNMVDKERQLHLKGKCHNCVYNLMGKREKKPGEFGVAKGSRTIWYMWLGSRFLEFE
ALGFLNEERWASRDISGGGVEGIGINYLGYHLEKMARKVGVLYADDTAGWDTRVTQADLEDERELLHFME
GEHKRLAEAIFKLTYENKVALCPRPGSKGGTVMDVISRKDQRGSGQVVTYALNTLTNIKVQLVRMAESEG
ILTPELEDLGIEQWLKQNGEDRLSRLLVSGDDCVVNALDERFGKALTWLNTMEKTRKDIEAWKPSRAYRT
WQEVEFCSHHFHELFLRDGRKLIVPCRDQHELVGRSMVSQGGSSGVAGTACLAKAYAQMWLMSYFHRRDL
RTLGFAIMSSVPKDWFPTGRTTWSVHANHEWLSNENILEIWNRVWIEENPFMEDKTTVAEWKNIPYLQRN
QDLSCSSLIDNPTRAQWAKLLKGAVMKVREMIGREHYSDYLSNMGRYQEGTEEFHMW
>REF_ccCFV.ncbi-refseqs.MK473878
GFGNVDLAGEQWKNALNQMSSSSFNKYRTHKVVERKKEQSVSRGAHKMDEIIRKFGWVPKGDVIDLGCGQ
GGWSQRMAVENEVDTVRGHTIGGTHAAPSSIKTYGHESITLFDHEDVFTKELSRKDTIVCDIGEQDNNWK
IEAQRTAKVIELVMRWLSFNPTARFCFKVLCPYDTNVMQRLRNLQRKYGGRLIRLTGSRNSNHEMYMISG
QSCSLINSVATTTIMLQNRMNENWKEKILLEDITYPEGEKSIVEPSFPPESAVKNRIKYIKATHPLIKHD
...
</code></pre>

		 <p>
		   To export to a file (rather than previewing on the console), use
		   the -f (--file) option in place of -p and specify a file name,
		   as follows:
		 </p>
		 
	   
<pre>
Mode path: /project/flaviviridae/module/fastaProteinExporter
GLUE> export reference -f NS5 -w "sequence.genus = 'Flavivirus'" -f flavivirus-ns5-aa.fasta
</code></pre>



	   <h3>
		 <a id="Example1Question1" class="anchor" href="#Example1Question1" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Question 1: How do I know what features or properties have been defined?</strong>
	   </h3>
	   <hr>


	   <h3>
		 <a id="Example1Question2" class="anchor" href="#Example1Question2" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Question 2: How do I export protein sequences for all sequences?</strong>
	   </h3>
	   <hr>



	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/">Flavivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



