<!DOCTYPE html>
<html lang="en-us">

  <head>


    <meta charset="UTF-8">
    <title>Flaviviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flaviviridae-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flaviviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/zipball/master" class="btn">Download</a>

      <a target="_blank" href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">





	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of flaviviruses using Flavivirus-GLUE</strong>
	   </h3>
	   <hr>
	   


	   <p>
		 <b>Flavivirus-GLUE</b> can be used to perform comparative genomic analysis of flaviviruses
		 in an <b>efficient, standardised and reproducible</b> way.
	   </p>

        
       <p><img src="../assets/images/flavivirus-header.jpg" alt="Flaviviruses"/></p>
         
	   

<blockquote>
		 <p>
		   Comparative genomic analysis of flaviviruses can help reveal the biological basis
		   of flavivirus replication.
		   The insights obtained via comparative analysis of flavivirus genomes can directly
		   inform efforts to limit flavivirus-associated disease and develop flavivirus-based 
		   therapeutics. 
		 </p>
         
</blockquote>


	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous flaviviral elements (EFVs)
		 can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>




	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Sequences and associated data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project are organised into multiple distinct sources.
		 Each source contains data in either GenBank XML or plain FASTA format.
		 The type of data is indicated by the name of the source (all GenBank XML sources 
		 contain 'ncbi' in the name).
	   </p>
	   

	   <p>
		 GenBank XML files are imported into this project directly from <a href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a> using
		 a GLUE module and are uniquely identified within this project by their <a target="_blank" href="https://www.ncbi.nlm.nih.gov/Sequin/acc.html">GenBank accession numbers</a>. 

		 Where possible, we prefer to use sequences obtained via GenBank since it
		 represents the principle source of published nucleotide sequence data.
		 However, FASTA sources can also be used in GLUE,
		 making it straightforward to expand the curated, GenBank-derived set with 
		 unpublished sequences. 
	   </p>



	   <p>
        Currently his project includes only one  NCBI-derived source
        - <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/sources/genus/amdo/ncbi-refseqs"><b>ncbi-refseqs</b></a>:
        Core project master reference sequences (one per flavivirus genus).		 		 
	   </p>

	   <p>
         Sequences included in this project are linked to
         <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/core/ncbi-refseqs-side-data.tsv"><b>auxiliary data in tabular format</b></a>.
	   </p>

	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Flavivirus reference sequences</strong>
	   </h3>
	   <hr>

		<p>

 		  For selected flaviviral genera, we defined a 'master' reference sequence, as follows:


	   <h4>
		 <a id="Genus Flavivirus" class="anchor" href="#Epiflavivirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Genus <i>Flavivirus</i></strong>
	   </h4>


		  <ul>
		    <li><b><i>Flavivirus Mosquito 2</i></b>: Yellow fever virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002031"><b>NC_002031</b></a>)</li>
		    <li><b><i>Flavivirus Mosquito 1</i></b>: Dengue virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001477"><b>NC_001477</b></a>)</li>
		    <li><b><i>Tick-borne Flaviviruses</i></b>: Powassan virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003687"><b>NC_003687</b></a>)</li>
		    <li><b><i>No-known vector group 1</i></b>: Apoi virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003676"><b>NC_003676</b></a>)</li>
		    <li><b><i>No-known vector group 2</i></b>: Sokuluk virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_026624"><b>NC_026624</b></a>)</li>
		    <li><b><i>dual-host insect-specific flavivirus group</i></b>: Lammi virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024806"><b>NC_024806</b></a>)</li>
		    <li><b><i>classical insect-specific flavivirus group</i></b>: Kamiti river virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_005064"><b>NC_005064</b></a>)</li>

		  </ul>

		</p>



	   <h4>
		 <a id="Other Flaviviridae genera" class="anchor" href="#Epiflavivirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Other <i>Flaviviridae</i> genera</strong>
	   </h4>


		  <ul>
		    <li><b><i>Tamanavirus</i></b>: Tamana bat virus 7 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Jingmenvirus</i></b>: Jingmen tick virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Pestivirus</i></b>: Bovine viral diarrhea virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Large genome flaviviruses 1</i></b>: Soybean cyst nematode virus 5  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024077"><b>NC_024077</b>)</a></li>
		    <li><b><i>Large genome flaviviruses 2</i></b>: Shuangao lacewing virus 2  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Hepacivirus</i></b>: Hepatitis C virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004102"><b>NC_004102</b>)</a></li>
		    <li><b><i>Pegivirus</i></b>: Human pegivirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_027998"><b>NC_027998</b>)</a></li>

		  </ul>


		<p>

          We explicitly defined the locations of genome features
          on master reference sequences using GLUE commands 
          (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/core/flaviviridaeExplicitFeatureLocations.glue">here</b></a>).
		  
 		</p>
        


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>


		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUEs <b>alignment tree</b> data structure - 
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained multiple sequence alignments representing different
          taxonomic levels, and we've used this approach in Flavivirus-GLUE.
 		</p>


 	    <br>

        <p><img src="../assets/images/flavi-msa-tree.jpg" alt="Alignment tree concept" /></p>
 		

 	    <br>


<blockquote>
		<p>
		  The schematic figure above shows the alignment tree structure in <b>Flavivirus-GLUE</b>.
		</p>
         
</blockquote>


		<p>       
          For the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.
          For the highest taxonomic levels (i.e. at the root) we aligned only the most
          conserved gene (the viral polymerase). 
          We used an alignment tree data structure to link these  alignments,
          via a set of common reference sequences.
          The root alignment contains all reference sequences, whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
 		</p>
             
   
		<p>
             
          
          Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/alignments/root/flavi-ns-root.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major flavivirus lineages</li>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/family-root"><strong>Family-level (root)</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/genus-level"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/">Flaviviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



