<!DOCTYPE html>
<html lang="en-us">

  <head>


    <meta charset="UTF-8">
    <title>Flavivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flavivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of flaviviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/zipball/master" class="btn">Download</a>

      <a target="_blank" href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">





	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of flaviviruses using Flavivirus-GLUE</strong>
	   </h3>
	   <hr>


	   <p>

		 Arthropod-borne flaviviruses such as Zika virus (<a target="_blank" href="https://www.cdc.gov/zika/index.html"><b>ZIKV</b></a>),
		 Dengue virus (<a target="_blank" href="https://www.cdc.gov/dengue/index.html"><b>DENV</b></a>), and
		 yellow fever virus (<a target="_blank" href="https://www.cdc.gov/yellowfever/index.html"><b>YFV</b></a>)
		 are the causative agents of large-scale outbreaks that result
		 in millions of human infections every year, while the bloodborne hepatitis C virus
		 (<a target="_blank" href="https://www.who.int/news-room/fact-sheets/detail/hepatitis-c"><b>HCV</b></a>)
		 is a major cause of chronic liver disease.


	   </p>



	   <br>


      
       <p><img src="../assets/images/flavivirus-header.jpg" alt="Flaviviruses"/></p>
         

<blockquote>
		 <p>
		   Projected urbanisation in 2027 (from <a target="blasnk" href="https://www.economist.com/node/21642053">The Economist</a> magazine).
		   The growth of mega-cities in tropical and semi-tropical regions presents a challenge
		   to controlling the spread of mosquito-borne flaviviruses.
		   Comparative genomic analysis can help reveal the biological basis
		   of flavivirus replication, directly informing efforts to limit flavivirus-associated disease.
		 </p>
         
</blockquote>


	   <br>


	   <p>
		 Comparative genomic approaches can be used to reveal the structural and functional
		 basis of flavivirus adaptations, such as the capacity to replicate in both arthropod
		 and vertebrate hosts. 
		 <b>Flavivirus-GLUE</b> can be used to perform comparative genomic analysis of flaviviruses
		 in an <b>efficient, standardised and reproducible</b> way.
	   </p>


	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous flaviviral elements (EFVs)
		 can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>


	   <br>



	   <p>
		   <b>NOTE</b>: those specifically interested in <b>hepatitis C virus (HCV)</b>
		   may want to investigate    
		   <a target="_blank" href="https://github.com/giffordlabcvr/HCV-GLUE"><b>HCV-GLUE</b></a>
		   and
		   <a target="_blank" href="https://github.com/giffordlabcvr/NCBI-HCV-GLUE"><b>NCBI-HCV-GLUE</b></a>.		   
		   
		   These GLUE projects were developed specifically for HCV and incorporate
		   a <b>graphical user interface (GUI)</b>
		   that allows users to browse and interrogate the underlying GLUE database
		   via 'point-and-click' methods.
		   The <b>MRC-University of Glasgow Centre for Virus Research</b> hosts an instance of the
		   <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>GUI version of HCV-GLUE</b></a>.
	   </p>


	   <br>




	   <h3>
		 <a id="sequenceData" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Sequences and associated data</strong>
	   </h3>
       <hr>


	   <p>
		 The sequence data in this project are organised into multiple distinct sources.
		 Each source contains data in either GenBank XML or plain FASTA format.
		 The type of data is indicated by the name of the source (all GenBank XML sources 
		 contain 'ncbi' in the name).
	   </p>
	   

	   <p>
		 GenBank XML files are imported into this project directly from
		 <a target="_blank" href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a> using
		 a GLUE module and are uniquely identified within this project by their
		 <a target="_blank" href="https://www.ncbi.nlm.nih.gov/Sequin/acc.html">GenBank accession numbers</a>. 

		 Where possible, we prefer to use sequences obtained via GenBank since it
		 represents the principle source of published nucleotide sequence data.
		 However, FASTA sources can also be used in GLUE,
		 making it straightforward to expand the curated, GenBank-derived set with 
		 unpublished sequences. 
	   </p>



	   <p>
        Currently this project includes only one NCBI-derived source
        - <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/sources/ncbi-refseqs"><b>ncbi-refseqs</b></a>:
        It contains the reference genome sequences (one per flavivirus species) used in this project.		 		 
	   </p>

	   <p>
         Sequences included in this project are linked to
         <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/tabular/core/flavi-ncbi-refseqs-side-data.tsv"><b>auxiliary data in tabular format</b></a>.
	   </p>



	   <br>



	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong><i>Flaviviridae</i> reference sequences</strong>
	   </h3>
	   <hr>

		<p>

 		  We defined a 'master' reference sequence to represent recognised flavivirus genera/subgenera, as follows:


	   <h4>
		 <a id="Genus Flavivirus" class="anchor" href="#Epiflavivirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Genus <i>Flavivirus</i></strong>
	   </h4>


		  <ul>
		    <li><b><i>Mosquito-borne flavivirus group 2</i></b>: Yellow fever virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_002031"><b>NC_002031</b></a>)</li>
		    <li><b><i>Mosquito-borne flavivirus group 1</i></b>: Dengue virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001477"><b>NC_001477</b></a>)</li>
		    <li><b><i>Tick-borne flaviviruses</i></b>: Powassan virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003687"><b>NC_003687</b></a>)</li>
		    <li><b><i>No-known vector group 1</i></b>: Apoi virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003676"><b>NC_003676</b></a>)</li>
		    <li><b><i>No-known vector group 2</i></b>: Sokuluk virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_026624"><b>NC_026624</b></a>)</li>
		    <li><b><i>Dual-host insect-specific flavivirus group</i></b>: Lammi virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024806"><b>NC_024806</b></a>)</li>
		    <li><b><i>Mpulungu flavivirus group</i></b>: Mpulungu flavivirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/LC582740"><b>LC582740</b></a>)</li>
		    <li><b><i>Classical insect-specific flavivirus group</i></b>: Kamiti river virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_005064"><b>NC_005064</b></a>)</li>
		    <li><b><i>Crustacean flavivirus group</i></b>: Crangon crangon flavivirus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MK473878"><b>MK473878</b></a>)</li>

		  </ul>

		</p>



	   <h4>
		 <a id="Other Flaviviridae genera" class="anchor" href="#Epiflavivirinae" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Other <i>Flaviviridae</i> genera</strong>
	   </h4>


		  <ul>
		    <li><b><i>Tamanavirus</i></b>: Tamana bat virus 7 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Jingmenvirus</i></b>: Jingmen tick virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Pestivirus</i></b>: Bovine viral diarrhea virus 1 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Pesti-like 1 (PL2)</i></b>: Soybean cyst nematode virus 5  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_024077"><b>NC_024077</b>)</a></li>
		    <li><b><i>Pesti-like 2 (PL2)</i></b>: Shuangao lacewing virus 2  (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_040562"><b>NC_040562</b>)</a></li>
		    <li><b><i>Hepacivirus</i></b>: Hepatitis C virus (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004102"><b>NC_004102</b>)</a></li>
		    <li><b><i>Pegivirus</i></b>: Human pegivirus 2 (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_027998"><b>NC_027998</b>)</a></li>

		  </ul>


		<p>

          We explicitly defined the locations of genome features
          on master reference sequences using GLUE commands 
          (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/glue/build/core/flaviviridaeExplicitFeatureLocations.glue">here</b></a>).
		  
 		</p>
        


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments (MSAs)</strong>
	   </h3>
	   <hr>

		<p>
          Multiple sequence alignments (MSAs) are the basic currency of comparative genomic analysis.
          MSAs constructed in this study are linked together using
          GLUE's <b>constrained MSA tree</b> data structure.
  		</p>

         
 		<p>
          A 'constrained MSA' is an alignment in which the coordinate space is defined by 
          a selected reference sequence. Where alignment members contain insertions relative
          to the reference sequence, the inserted sequences are recorded and stored
          (i.e. sequence data is never deleted).
  		</p>

         
 		<p>
           
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained MSAs representing different taxonomic levels,
          and we've used this approach in Flavivirus-GLUE.
 		</p>


 	    <br>

        <p><img src="../assets/images/flavi-msa-tree.jpg" alt="Alignment tree concept" /></p>
 		

<blockquote>
		<p>
		  The schematic figure above shows the '<b>alignment tree</b>' data structure 
		  currently implemented in <b>Flavivirus-GLUE</b>.
          For the highest taxonomic levels (i.e. at the root) we aligned only the most
          conserved regions of the genome, whereas for the lower
          taxonomic levels (i.e. within and below genus level) we aligned complete coding
          sequences.

          We used an alignment tree data structure to link these alignments,
          via a set of common reference sequences.
          The root alignment contains reference sequences for major clades,
          whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
		</p>
         
</blockquote>


 	    <br>

            
   
		<p>
             
          
          Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/blob/master/alignments/root/flaviviridae-root.fna"><strong>root</strong></a>’
			alignment (i.e. family-level) constructed to represent homology between the two largest subgroupings in the <i>Flaviviridae</i>.
			
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/major-lineage"><strong>major-lineage</strong></a>’
			alignments constructed to represent proposed homologies between representative members
			of major <i>Flaviviridae</i> lineages.</li>

			<li> ‘<a target="_blank" href="hhttps://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/minor-lineage"><strong>minor-lineage</strong></a>’
			alignments constructed to represent proposed homologies between representative members
			of 'minor' <i>Flaviviridae</i> lineages.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/genus/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirus genera.</li>
			
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/alignments/internal/subgenus/"><strong>subgenus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirus subgenera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
          
		  <ol>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/major-lineage"><strong>major-lineage</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of major flavivirus lineages.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/genus"><strong>minor-lineages</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of minor flavivirus genera.</li>

			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/genus"><strong>genus-level</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of specific flavivirus genera.</li>
			
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Flavivirus-GLUE/tree/master/trees/subgenus"><strong>subgenus-level</strong></a>’
			phylogenetic trees showing reconstructed evolutionary relationships between 
			representative members of specific flavivirus subgenera.</li>
		  </ol>

 		</p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/">Flaviviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



