<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Flaviviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Flaviviridae-GLUE</h1>
      <h2 class="project-tagline">A sequence data resource for flaviviruses.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous flavivirus (EVE) data</strong>
	   </h3>

		<p>
          We have used GLUE to organise the fossil record of flaviviruses.
          You can access these data
          <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master">in this repository</a>. 
          This website provides quick links to specific data items. 

		</p>


	   <h3>
		 <a id="EVE reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EVE reference sequences and data</strong>
	   </h3>

		<p>
           We constructed reference sequences for EVEs using alignments of EVE
           sequences derived from the same initial germline colonisation event - i.e.
           orthologous elements in distinct species, and paralogous 
           elements that have arisen via intragenomic duplication of EVE sequences.
  		</p>




	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>

		<p>

          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          ‘<a href="./alignment tree concept small.png">alignment tree</a>’
          data structure. Alignments in the project include:

		  <ol>
			<li> A single ‘<a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/blob/master/alignments/root/Flaviviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major flavivirus lineages
			(including extinct lineages represented only by EVEs).</li>
			<li> ‘<a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master/alignments/internal/"><strong>Genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific flavivirus genera and EVE reference sequences.
			<li> ‘<a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master/alignments/tip"><strong>Tip</strong></a>’
			alignments in which all taxa are derived from a single EVE lineage.</li>
		  </ol>


 		</p>


	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>

		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master/trees/main-root"><strong>root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master/trees/eve-genus"><strong>Genus-level</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/tree/master/trees/eve-orthologs"><strong>EVE lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>




	   <h3>
		 <a id="Raw EVE sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw EVE sequences and data</strong>
	   </h3>

		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The tabular files contain information about the genomic location of each EVE. 
          EVEs were classified by comparison to a polypeptide sequence reference library
          designed to represent the known diversity of flaviviruses - this includes extinct 
          lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>


		<p>
           Raw data in tabular format are <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/blob/master/tabular/eve/efv-side-data.tsv"><strong>here</strong></a>.
  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files) are <a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/blob/master/tabular/efv-refseqs/efv-refseq-side-data.tsv"><strong>here</strong></a>.
  		</p>
		

	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for EVEs</strong>
	   </h3>

		<p>
          We have applied a systematic approach to naming EVE, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>

		<p>
          The first component is the classifier ‘EVE’ (endogenous hepatitis B virus/endogenous flavivirus). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of EVE group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
		</p>
		
		<p>
          The third component of the ID defines the set of host species in which the ortholog occurs.
 		</p>



	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		<p>
		
		
		Lytras S, Arriagada G, and RJ Gifford
		<b>(2020)</b>
		<br>
		Ancient evolution of flaviviral paleoviruses and their impact on host genomes.
		<br>
		<a href="https://www.biorxiv.org/content/10.1101/2020.11.02.364562v1" target="_blank">[Preprint]</a>
		<br>
		<br>

		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		Zhu H, Dennis T, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		<a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		<br>
		<br>

		Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		<b>(2018)</b>
		<br>
		Nomenclature for endogenous retrovirus (ERV) loci.
		<br>
		<i>Retrovirology</i>
		<a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		<br>
		<br>

		Katzourakis A. and RJ. Gifford
		<b>(2010)</b>
		<br>
		Endogenous viral elements in animal genomes.
		<br>
		<i>PLoS Genetics</i>
		<a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Flaviviridae-GLUE/">Flaviviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



